README.md

```diff
@@ -9,13 +9,23 @@ The AWS Toolkit for Visual Studio Code is an extension for working with AWS serv
 
 The Toolkit is available from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/itemdetails?itemName=AmazonWebServices.aws-toolkit-vscode).
 
-This is an open source project because we want you to be involved. We love issues, feature requests, code reviews, pull requests or any positive contribution.
+This is an open source project because we want you to be involved. We love issues, feature requests, code reviews, pull 
+requests or any positive contribution. See [CONTRIBUTING](CONTRIBUTING.md) for how to help.
 
 ## Documentation
 
 -   The [Quick Start Guide](extension-readme.md) provides a summary of the Toolkit and the most common tasks.
 -   The [User Guide](https://docs.aws.amazon.com/console/toolkit-for-vscode/welcome) contains detailed instructions for getting up and running with the Toolkit.
 
+## Feedback
+
+We want your feedback!
+
+- Upvote 👍 [feature requests](https://github.com/aws/aws-toolkit-vscode/issues?q=is%3Aissue+is%3Aopen+label%3Afeature-request+sort%3Areactions-%2B1-desc)
+- Ask a question on [Stack Overflow](https://stackoverflow.com/questions/tagged/aws-toolkit)
+- [Request a new feature](./CONTRIBUTING.md#reporting-bugsfeature-requests)
+- [File an issue](https://github.com/aws/aws-toolkit-vscode/issues)
```

Comment:
nit: why is one a link to contributing and one a direct link to issues if the contributing section is about both?

Reply:
updated  to point to issue templates